torch/nn/modules/normalization.py

```diff
@@ -174,7 +176,10 @@ def __init__(self, normalized_shape: _shape_t, eps: float = 1e-5, elementwise_af
         self.elementwise_affine = elementwise_affine
         if self.elementwise_affine:
             self.weight = Parameter(torch.empty(self.normalized_shape, **factory_kwargs))
-            self.bias = Parameter(torch.empty(self.normalized_shape, **factory_kwargs))
+            if bias:
+                self.bias = Parameter(torch.empty(self.normalized_shape, **factory_kwargs))
+            else:
+                self.register_parameter('bias', None)
```

Comment:
Curious: Why register `"bias"` as `None`? This seems to set `_parameters["bias"] = None`.
https://github.com/pytorch/pytorch/blob/783a46adeeadf40b88f7450fe23daf5a7a4bb526/torch/nn/modules/module.py#L570-L571
However, we can still later set `.bias = nn.Parameter(...)` without error, so I am curious what the point of calling `register_parameter("bias", None)` is.

Reply:
This ensures that the name is reserved for a Parameter and make the Module more consistent with and without the arg passed. We do this for linear/conv as well.